pandas/tests/reshape/concat/test_concat.py

```diff
@@ -507,7 +507,7 @@ def test_concat_duplicate_indices_raise(self):
             concat([df1, df2], axis=1)
 
 
-@pytest.mark.parametrize("dt", np.sctypes["float"])
+@pytest.mark.parametrize("dt", np.core.sctypes["float"])
```

Comment:
If it's not good practice to import from `np.core`, could you just replace this with the `float_numpy_dtype` fixture?